app/src/main/java/tech/relaycorp/letro/ui/theme/Type.kt

```diff
@@ -38,37 +38,37 @@ val Typography = Typography(
     headlineSmall = TextStyle(
         fontFamily = Inter,
         fontWeight = FontWeight.SemiBold,
-        fontSize = 24.sp,
-        lineHeight = 32.sp,
-        letterSpacing = (-0.6).sp,
+        fontSize = 22.sp,
```

Comment:
Can we please change it to:

fontWeight = FontWeight.SemiBold,
fontSize = 24.sp,
lineHeight = 30.sp,
letterSpacing = (-0.25).sp,